src/to_mathlib/analysis/inner_product_space/tensor_product_of_inner_product_finite_dimensional.lean

```diff
@@ -0,0 +1,412 @@
+import to_mathlib.analysis.inner_product_space.tensor_product
+import analysis.inner_product_space.gram_schmidt_ortho
+import data.fin.basic
+
+noncomputable theory
+
+open_locale real_inner_product_space tensor_product big_operators
+
+namespace inner_product_space
+
+variables (E F : Type*) [linear_order E] [linear_order F]
```

Comment:
We definitely don't want these assumptions!
```suggestion
variables (E F : Type*)
```

Reply:
has been removed